uiowa/uiowa
# Feature: LB styles regression for containers

Resolves https://github.com/uiowa/uiowa/issues/2324

# How to test

1. Test on the home page for admissions.uiowa.edu and verify that space above image can be removed by adding the "Remove top spacing" option in the layout builder styles. 
2. Test on https://diversity.uiowa.edu/dei-action-planning and verify that there is `1.25rem` of space between the page title and sidebar layout. 
3. Test on uiowa.edu, Collegiate and v2 sites and verify that vertical spacing is not missing or extra spacing has been added.
